votes_up.php

```diff
@@ -8,90 +8,103 @@
 */
 
 
-//----------------------------------------------------------
-define( 'II_VERSION', '1.0.0' );
-define( 'II__PLUGIN_URL', plugin_dir_url( __FILE__ ) );
-define( 'II__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
-
-require_once 'votes_up_admin.php';
-
-
-register_activation_hook(__FILE__,'votes_up_create_table');
-
-$ii_db_version = "1.0";
-
-//создание пользовательской таблицы
-function votes_up_create_table () {
-    global $wpdb;
-    global $ii_db_version;
-    require_once(ABSPATH . 'wp-admin/includes/upgrade.php');
-
-    $table_name = $wpdb->prefix . "votes_up";
-    if($wpdb->get_var("SHOW TABLES LIKE '$table_name'") != $table_name) {
-        $sql = "CREATE TABLE " . $table_name . " (
-	        id mediumint(9) KEY NOT NULL AUTO_INCREMENT,
-	         votes_up_cred DOUBLE(5,2) DEFAULT '0' NOT NULL,
-	         choice TINYINT(1) DEFAULT '0' NOT NULL,
-	         Pay_message TINYINT(1) DEFAULT '0' NOT NULL,
-	         pay_vote_cred DOUBLE(5,2) DEFAULT '0' NOT NULL,
-	         Pay_reply TINYINT(1) DEFAULT '0' NOT NULL
-	        );";
-
-        dbDelta($sql);
-
-         $wpdb->insert( $table_name, array(
-             'votes_up_cred' => 0,
-             'choice' => 0,
-             'Pay_message' => 0,
-             'pay_vote_cred' => 0,
-             'Pay_reply' => 0
-         ));
-    }
-    add_option("ii_db_version", $ii_db_version);
-}
+// Hook for adding admin menus
+add_action('bbp_forum_metabox', 'add_pay_topic');
+add_action('bbp_topic_metabox', 'add_pay_reply');
+// Hook for adding  attributes
+add_action( 'save_post', 'add_forum_attributes' );
+add_action( 'save_post', 'add_reply_attributes' );
+// Add custom fields to bbpress topics and replyes on front end
+add_action( 'bbp_theme_before_reply_form_notices', 'theme_before_reply' );
+add_action ( 'bbp_theme_before_topic_form_notices', 'theme_before_topic');
+//Remove creds
+add_action('init', 'pay_for_topic');
+add_action('init', 'pay_for_reply');
+//Add creds for vote
+add_action('bbpvotes_do_post_vote', 'do_post_vote_up',10,1);
 
+// функция добавления чекбокса в свойства форума    \/
```

Comment:
All comments should be in English
Every function should have correct PHPDoc comments generated